src/NzbDrone.Core/Indexers/Definitions/BeyondHD.cs

```diff
@@ -190,11 +190,11 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                 throw new IndexerException(indexerResponse, $"Unexpected response status {indexerResponse.HttpResponse.StatusCode} code from API request");
             }
 
-            // TODO Have BHD fix their API response content type so we can proper check here
-            // if (!indexerResponse.HttpResponse.Headers.ContentType.Contains(HttpAccept.Json.Value))
-            // {
-            //     throw new IndexerException(indexerResponse, $"Unexpected response header {indexerResponse.HttpResponse.Headers.ContentType} from API request, expected {HttpAccept.Json.Value}");
-            // }
+            if (!indexerResponse.HttpResponse.Headers.ContentType.Contains(HttpAccept.Json.Value))
```

Comment:
For Posterity: Forum post confirming this is now fixed so this is okay to be included